### PR TITLE
Add JSON logging instructions [MC-2844]

### PR DESCRIPTION
**Pull request by emrberk**

### CODE REVIEW

File: docs/modules/deploy-manage/pages/logging.adoc [hunk 1]
```diff
@@ -80,6 +80,8 @@ You can change the logging template by providing a JSON logging template file as
 ----
 -Dlogging.json.template=<json-file-path>
 ----
+NOTE: For a provided JSON logging template to be effective, you need to activate the `json-logging` profile.
```
Comment (rebekah-lawrence) [inline docs/modules/deploy-manage/pages/logging.adoc:83]:
```suggestion
NOTE: To use a custom JSON logging template, you must activate the `json-logging` profile as described above.
```

**Review by promanenko — APPROVED** [review]

File: docs/modules/deploy-manage/pages/logging.adoc (hunk 1 above)
Comment (rebekah-lawrence) [reply docs/modules/deploy-manage/pages/logging.adoc:83]: I am assuming that it is this instruction that needs to be followed:

> Management Center provides JSON template logging out of the box. To enable JSON logging, you can activate json-logging profile by starting Management Center with the following parameter:

However, this is a bit unclear. If MC provides JSON logging OOTB, why do you need to enable it? Do we mean:

> To use a custom JSON logging template, you must activate the json-logging profile as follows: 
> 
>` Dspring.profiles.active=json-logging`

If so, the note is no longer required.

**Review by rebekah-lawrence — CHANGES_REQUESTED** [review]

File: docs/modules/deploy-manage/pages/logging.adoc (hunk 1 above)
Comment (emrberk) [reply docs/modules/deploy-manage/pages/logging.adoc:83]: Hi Rebekah, we were following the instructions in https://hazelcast.slack.com/archives/C6YCTD1RA/p1710281406856269 to enable JSON logging in MC. The recent change made it possible to enable it only activating a profile, that's why we use `out of the box`. 
Providing JSON template layout however is only for changing the structure of the JSON logs. So if you don't activate `json-logging` profile, providing a JSON template has no effect, as the logs won't be in JSON format.
We added the note to remove possible confusions in a situation like: not activating the JSON logging profile, but passing a template, and expecting it to  be effective.

Comment (rebekah-lawrence) [reply docs/modules/deploy-manage/pages/logging.adoc]: Ahhhh, OK. In that case, I'd suggest a slight change to the OOTB statement:

Hazelcast Platform provides JSON template logging out of the box. However, to enable JSON template logging in Management Centre you must activate a profile. To activate a json-logging profile, start Management Center with the following parameter:

Comment (emrberk) [reply docs/modules/deploy-manage/pages/logging.adoc]: Is there a specific reason about mentioning Platform capabilities here? 😅 
The flow in Platform and MC is nearly same right now (configured through two parameters). I think usage of "However" would be misleading here.

Comment (rebekah-lawrence) [reply docs/modules/deploy-manage/pages/logging.adoc]: I was looking at the Slack conversation you linked, where it says:

> Although Hazelcast Platform supports this out-of-the-box, MC requires some additional configuration.

It is just not logical to say that something is provided/supported OOTB when you still have to enable it. OOTB means no further config required, it just works.

Maybe just remove the first sentence from the existing version and start "To enable....."?

Comment (emrberk) [reply docs/modules/deploy-manage/pages/logging.adoc]: It makes sense, I removed the first sentence 👍

**Review by rebekah-lawrence — APPROVED** [review]
Thanks for bearing with me ;-)

**Comment by emrberk** [comment]
@rebekah-lawrence I guess the test failures are about Platform. Do we have an action item here, or can I merge this one?

**Comment by rebekah-lawrence** [comment]
@emrberk Merging this should be fine. It will be on the SNAPSHOT branch; so pre-release information, which can still change. If any further changes are required after a solution to the test failures, we can handle it then.

**Pull request merged**